libs/core/src/lib/combobox/combobox.component.ts

```diff
@@ -368,10 +407,12 @@ export class ComboboxComponent implements ControlValueAccessor, OnInit, OnChange
             this.open = isOpen;
             this.openChange.emit(this.open);
             this.onTouched();
-            if (isOpen) {
+            if (isOpen && this.displayedValues && this.displayedValues.length) {
```

Comment:
I see this check in 3 places. Wouldn't it be better if we move the logic to a function?